osf_tests/management_commands/test_migrate_employment_education_info.py

```diff
@@ -0,0 +1,66 @@
+# encoding: utf-8
+import json
+import pytest
+
+from osf_tests.factories import UserFactory
+from osf.models import UserEducation, UserEmployment
+from django.db import connection
+from osf.management.commands.migrate_education_employment import populate_new_models
+
+
+@pytest.mark.django_db
+class TestEducationEmploymentInfoMigration:
+
+    @pytest.fixture()
+    def user(self):
+        user = UserFactory()
+        user.save()
+        schools = [
+            {
+                u'degree': u'BA Communications',
+                u'department': u'Communications',
+                u'endMonth': None,
+                u'endYear': 1996,
+                u'institution': u'Clemson',
+                u'ongoing': False,
+                u'startMonth': None,
+                u'startYear': 1973
+            }
+        ]
+        jobs = [
+            {
+                u'title': u'Safety',
+                u'department': u'Defense',
+                u'endMonth': None,
+                u'endYear': 2008,
+                u'institution': u'Philadelphia Eagles',
+                u'ongoing': False,
+                u'startMonth': None,
+                u'startYear': 1996
+            }, {
+                u'title': u'Hall of Fame Safety',
+                u'department': u'Defense',
+                u'institution': u'Philadelphia Eagles',
+                u'ongoing': True,
+                u'startYear': 2018
+            }
+        ]
```

Comment:
Please include the new failure case into tests, and would it be worth coming up with a few more potential bad data scenarios based on the constraints (or lack thereof) on the data, historical and current?